README.md

```diff
@@ -0,0 +1,251 @@
+
+<a name="spring-boot"></a>
+### Spring Boot
+
+1. Spring Boot makes it easy to create stand-alone, production-grade Spring based Applications that you can "just run".
+Some features and advantages are:
+
+    * Create stand-alone Spring applications.
+    * Embed Tomcat, Jetty or Undertow directly (no need to deploy WAR files)
+    * Provide opinionated 'starter' dependencies to simplify your build configuration
+    * Automatically configure Spring and 3rd party libraries whenever possible
+    * Provide production-ready features such as metrics, health checks, and externalized configuration
+    * Absolutely no code generation and no requirement for XML configuration
+
+Source: [https://spring.io/projects/spring-boot](https://spring.io/projects/spring-boot)
+
+
+<!-- ************************* -->
+<!--    DIGITAX API SECTION    -->
+<!-- ************************* -->
+## DIGITAX API
+The base URL for DIGITAX: http://52.6.241.241/.
+
+### Sign Up
+Endpoint: `/auth/signup`
+
+API endpoint for user registration.
+
+| Code | Description              | Links    |
+|------|--------------------------|----------|
+| 200  | CREATE_SUCCESS           | No Links |
+| 400  | USER_ALREADY_EXISTS      | No Links |
+| 401  | INVALID_EMAIL_FORMAT     | No Links |
+| 401  | INVALID_PASSWORD_FORMAT  | No Links |
+| 500  | INTERNAL_SERVER_ERROR    | No Links |
+
+<details>
+    <summary>POST Request Body: </summary>
+
+```json
+{
+    "email": "example@test.com",
+    "password": "examplePass11"
+}
+```
+</details>
+<details>
+    <summary>Response:</summary>
+
+```json
+{
+  "user": {
+    "id": "97",
+    "phone": "555-555-5555",
+    "email": "example@test.com",
+    "userTypeId": "3",
+    "name": "John Doe",
+    "firstName": "John",
+    "lastName": "Doe",
+    "dateOfBirth": "01011983",
+    "sex": "male",
+    "verifiedEmail": "0",
+    "verifiedPhone": "0",
+    "active": "1",
+    "archived": "0",
+    "created": "2018-10-01 03:24:43",
+    "updated": "2018-10-01 03:24:43",
+    "deleted": "0"
+  },
+  "session": {
+    "accessToken": "07ywyJ6TdsMLOSh2GTfq328VyYEg6C3L",
+    "expirationMinutes": 300,
+    "UTCExpirationTime": "2019-10-16 18:52:27",
+    "PTExpirationTime": "2019-10-16 18:52:27"
+  }
+}
+```
+</details>
+
+[table of contents](#table-of-contents)
+
+### Sign In 
+Endpoint: `/auth/signin`
+
+API endpoint for user authentication.
+
+| Code | Description             | Links    |
+|------|-------------------------|----------|
+| 200  | SUCCESS                 | No Links |
+| 401  | WRONG_EMAIL_OR_PASSWORD | No Links |
+| 401  | INVALID_EMAIL_FORMAT    | No Links |
+| 401  | INVALID_PASSWORD_FORMAT | No Links |
+
+<details>
+    <summary>POST Request Body:</summary>
+
+```json
+{
+  "email": "example@test.com",
+  "password": "examplePass11"
+}
+```
+</details>
+<details>
+    <summary>Response:</summary>
+
+```json
+{
+  "user": {
```

Comment:
@Jayanta-Redapple Same here. When the `User` object is settled, update the response body in the README accordingly.